src/components/WizardPattern/__mocks__/wizardPatternExample.js

```diff
@@ -0,0 +1,150 @@
+  }
+  open() {
+    this.setState({ showModal: true });
+    setTimeout(() => {
+      this.setState({ wizardLoaded: true });
+    }, 1000);
+  }
+  renderStateless() {
+    return (
+      <div>
+        <Button bsStyle="primary" bsSize="large" onClick={this.open}>
+          Launch Stateless Modal Wizard <br />
+          (WizardPattern)
+        </Button>
+
+        <WizardPattern
+          show={this.state.showModal}
+          onHide={this.close}
+          onExited={this.close}
+          title="Modal Wizard Example"
+          shouldDisableNextStep={activeStepIndex => false}
+          steps={wizardSteps}
+          loadingTitle="Loading..."
+          loadingMessage="This may take a minute."
+          loaded={this.state.wizardLoaded}
+          activeStepIndex={this.state.activeStepIndex}
+          onStepChanged={activeStepIndex => this.setState({ activeStepIndex })}
+        />
+      </div>
+    );
+  }
+  renderStateful() {
+    return (
+      <div>
+        <Button bsStyle="primary" bsSize="large" onClick={this.open}>
+          Launch Stateful Modal Wizard <br />
+          (StatefulWizardPattern or WizardPattern.Stateful)
+        </Button>
+
+        <WizardPattern.Stateful
+          show={this.state.showModal}
+          onHide={this.close}
+          onExited={this.close}
+          title="Modal Wizard Example"
+          shouldDisableNextStep={activeStepIndex => false}
+          steps={wizardSteps}
+          loadingTitle="Loading..."
+          loadingMessage="This may take a minute."
+          loaded={this.state.wizardLoaded}
+        />
+      </div>
+    );
+  }
+  render() {
+    return this.props.stateful ? this.renderStateful() : this.renderStateless();
+  }
+}
+
+WizardPatternExample.propTypes = {
+  stateful: PropTypes.bool
+};
+
+WizardPatternExample.defaultProps = {
+  stateful: false
+};
+
+export const wizardPatternExampleSource = `
+  <Button bsStyle="primary" bsSize="large" onClick={this.open}>
+    Launch Stateless Modal Wizard <br />
+    (WizardPattern)
+  </Button>
+
+  <WizardPattern
+    show={this.state.showModal}
+    onHide={this.close}
+    onExited={this.close}
+    title="Wizard Pattern Example"
+    shouldDisableNextStep={activeStepIndex => false}
+    steps={[
+      {
+        title: 'General',
+        render: () => <p>(Step 1 Contents Here)</p>
+      },
+      {
+        title: 'Details',
+        render: () => <p>(Step 2 Contents Here)</p>
+      },
+      {
+        title: 'Results',
+        render: () => <p>(Step 3 Contents Here)</p>
+      }
+    ]}
+    loadingTitle="Loading..."
+    loadingMessage="This may take a minute."
+    loaded={this.state.wizardLoaded}
+    activeStepIndex={this.state.activeStepIndex}
+    onStepChanged={activeStepIndex => this.setState({ activeStepIndex })}
+  />
+`;
+
+export const statefulWizardPatternExampleSource = `
+  <Button bsStyle="primary" bsSize="large" onClick={this.open}>
+    Launch Stateful Modal Wizard <br />
```

Comment:
This is a pattern I am continuing only because in other places, if we don't have the raw string copy like this, the JSX doesn't quite render as we want it to in storybook. I haven't tried just using the JSX in this particular one. Modeled after existing files.